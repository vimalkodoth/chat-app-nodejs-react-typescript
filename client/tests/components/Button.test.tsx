import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../src/components/Button';

describe('Button', () => {
    it('should render component', async () => {
        const wrapper = shallow(<Button title="" />);
        expect(wrapper.getElements()).toMatchSnapshot();
    });
    it('should render a button', async () => {
        const wrapper = shallow(<Button title="button" />);
        expect(wrapper.find('button').exists()).toBeTruthy();
    });
});
