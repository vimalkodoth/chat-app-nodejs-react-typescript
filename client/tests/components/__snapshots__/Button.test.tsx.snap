// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render component 1`] = `
Array [
  <button>
    
  </button>,
]
`;
